flepimop/gempyor_pkg/src/gempyor/outcomes.py

```diff
@@ -18,40 +19,33 @@
 
 def run_parallel_outcomes(
     modinf: model_info.ModelInfo, *, sim_id2write, nslots=1, n_jobs=1
-):
+) -> Literal[1]:
     start = time.monotonic()
-
     sim_id2writes = np.arange(sim_id2write, sim_id2write + modinf.nslots)
-
-    loaded_values = None
+    random_seeds = np.random.choice(
+        range(nslots + 1, 4 * nslots), size=nslots, replace=False
+    ).tolist()
```

Comment:
confirming:
 - this is ensuring we have different seeds across different slots
 - these seeds are themselves a random draw, with no way to ensure a particular draw
 - so we are guaranteeing different outcome draws for different invocations?

Reply:
can leave adding seed option for future issue generally addressing matching more broadly - might be via generators anyway, so :shrug: